crates/bevy_internal/Cargo.toml

```
@@ -45,7 +45,7 @@ zlib = ["bevy_image/zlib"]
 zstd = ["bevy_image/zstd"]
 
 # Image format support (PNG enabled by default)
-avif = ["bevy_image/avif"]
+avif = ["bevy_image/avif-native"]
```

```suggestion
avif = ["bevy_image/avif"]
```
We need to enable the `bevy_image` feature flag here, which is still just called `avif`.